sdk/monitor/monitor-ingestion/src/logsIngestionClient.ts

```diff
@@ -77,27 +76,15 @@ export class LogsIngestionClient {
           contentEncoding: "gzip",
         });
       } catch (e: any) {
+        if (options?.errorCallback) {
+          options.errorCallback({ failedLogs: eachChunk, cause: isError(e) ? e : new Error(e) });
```

Comment:
When user implements the error callback, they have to use abortController.abort() and pass abortController as options to upload function